All EIPIP Meetings/Meeting 065.md

```diff
@@ -0,0 +1,472 @@
+
+**Pandapip**
+* So the particular patent waiver I added is basically the authors. If they have any patents, that they use in the EIPs, then they aren't allowed to sue anyone for using the EIPs. 
+
+**Golvin**
+* Well, they should disclose them Right now. 
+
+**Pandapip**
+* They're not obligated to though. 
+
+**Golvin**
+* That's true. 
+
+**Pandapip**
+* So, This basically just means that they don't, they can't patent troll. It has to be someone else, a third party. 
+
+**Golvin**
+* We, yeah, we specifically, we specifically say that, that we have nothing to do with them. so I'm generally in favor of just leaving it as it is. I hate putting more requirements on authors and we haven't had any problem yet. 
+
+**Victor**
+* I feel similar with Greg. I, I can see the value of, requiring patent to be granted or be disclosed. I feel it's a little bit too, too much to ask. And also, you can't prevent someone who propose an EIPs and influence someone else's patent, if they're not aware of. So I think it's general. 
+
+**Pandapip**
+* That's what I'm, what I'm suggesting is that basically the bat wave I've added is that the authors can't sue people for, for their own patents. Other people can other, anytime that it's a patent not owned by the author, it's like, say like some, some random third party, they use some random third party, image image, thing that's, that is, patented. The authors had no idea, patent waiver wouldn't apply there, since it's specifically the authors is authors is, right set are waved. Not those of the third party.
+
+**Victor**
+* I'm in, I'm in favor of, have it optional and make it a convenient way for people to wave the, the patent. because as author, if, if it's a sincere author, the easiest it, it is for the adopters, the better for the EIPs. So any sincere author would try to make it as easy as possible. But I, coming from an industry that cares a lot about patent and EIPss, for like, for, for people like me, I'm, I'm not even sure if I were able to contribute to EIPs anymore if there is a mandate for patent granting because of, other type, agreements that I have with, for example, like my employer. So that, that's the background. So I, I when, when Greg mentioned that, in Oracle employees were not even permitted to make, patent search, I, I totally resonate because that's some other trainings I've seen as well. So I, I think we're entering a space of legal practice and, none of us, I, I assuming is a lawyer here on this call. I, I can see the value and a good heart behind it. but I, I think we, we need a little bit more, We might need a little bit more expertise, in the, in a call for, for making it such a strong decision. I, I think optional is definitely valuable. Like making it optional I think is definitely valuable to be, 
```

Comment:
```suggestion
* I'm in favor of, have it optional and make it a convenient way for people to wave the, the patent. because as author, if, if it's a sincere author, the easiest it, it is for the adopters, the better for the EIPs. So any sincere author would try to make it as easy as possible. But I, coming from an industry that cares a lot about patent and EIPs, for like, for, for people like me, I'm, I'm not even sure if I were able to contribute to EIPs anymore if there is a mandate for patent granting because of, other type, agreements that I have with, for example, like my employer. So that, that's the background. So, I when, when Greg mentioned that, in Oracle employees were not even permitted to make, patent search, I, I totally resonate because that's some other trainings I've seen as well. So I, I think we're entering a space of legal practice and, none of us, I, I assuming is a lawyer here on this call. I, I can see the value and a good heart behind it. but I, I think we, we need a little bit more, We might need a little bit more expertise, in the, in a call for, for making it such a strong decision. I, I think optional is definitely valuable. Like making it optional I think is definitely valuable to be, 
```